doc/getting_started/getting_help.rst

```diff
@@ -18,3 +18,4 @@ it can be helpful to also browse the `Vega-Lite documentation <https://vega.gith
 .. _Vega-Lite: http://vega.github.io/vega-lite
 .. _bugs and feature requests: https://github.com/altair-viz/altair/issues/new/choose
 .. _StackOverflow: https://stackoverflow.com/tags/altair
+.. _Vega-Altair Google Group: https://groups.google.com/forum/#!forum/altair-viz
```

Comment:
```suggestion
```

I don't think any of us is monitoring this and I would prefer to have everything in as few places as possible so only directing users to GitHub and SO.